Dockerfile

```diff
@@ -25,15 +20,22 @@ RUN pip install --upgrade pip
 
 # Copy requirements.txt file and install requirements here to save time
 # when building this docker image again if no requirements have changed
-COPY $ADAGE_SRC/requirements.txt requirements.txt
+COPY adage/requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 
 # Clean up APT when done.
 RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Code to server directory
-COPY $ADAGE_SRC .
+# Copy necessary data files to bootstrap database
+COPY data data
+
+# Copy code for Django project
+COPY adage adage
+WORKDIR adage
+
+RUN wget https://bitbucket.org/greenelab/get_pseudomonas/raw/tip/get_pseudo_sdrf.py && \
+    wget https://bitbucket.org/greenelab/get_pseudomonas/raw/tip/gen_spreadsheets.py
```

Comment:
Yep - actually these lines were not in the original Dockerfile before this PR, I was putting them in because I was also replicating the current deployment in the fabfile (and these files are needed for deployment). For sure, I will  add a comment for these lines.